src/plusplus.coffee

```diff
@@ -142,7 +142,7 @@ module.exports = (robot) ->
     reasons = scoreKeeper.reasonsForUser(name)
 
     reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
-                     "#{name} has #{score} points. here are some raisins:" +
```

Comment:
I think it is a futurama reffrence. With Fry and the floating brains. "They
left for unknown raisins"

On Mon, Feb 22, 2016, 12:48 AM Mark Janssen <notifications@github.com>
wrote:

> In src/plusplus.coffee
> <https://github.com/ajacksified/hubot-plusplus/pull/59#discussion_r53603542>
> :
>
> > @@ -142,7 +142,7 @@ module.exports = (robot) ->
> >      reasons = scoreKeeper.reasonsForUser(name)
> >
> >      reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
> > -                     "#{name} has #{score} points. here are some raisins:" +
>
> It seems like a mistake a non-native speaker could make, since 'raisins'
> and 'reasons' have similar pronunciation. I do wonder why it has been in
> there for that long.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ajacksified/hubot-plusplus/pull/59/files#r53603542>.
>
-- 
Cheers,
Justin Presler
Facilities Supervisor
air.907.230.3698 land.907.258.2407